.circleci/config.yml

```diff
@@ -152,8 +152,13 @@ jobs:
           name: Authenticate with registry
           command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/fun/.npmrc
       - run:
-          name: Publish package
-          command: npm publish ./lib/
+          name: Copy project README into package
+          command: cp README.md ./lib/
+      - run:
+          name: Publish package as public
+          command: |
+            npm config set access public
+            npm publish ./lib/
```

Comment:
We can one-line it
`npm publish ./lib/ --access public`

Reply:
Good idea !